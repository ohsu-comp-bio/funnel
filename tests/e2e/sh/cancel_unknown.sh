# Test canceling a task that doesn't exist
funnel task cancel foo
