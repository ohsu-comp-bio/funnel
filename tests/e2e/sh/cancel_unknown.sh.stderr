Error: [STATUS CODE - 404]	{"error":"not found: taskID: foo","code":5}
